### PR TITLE
Group indicators by theme

### LINKED ISSUE #60: Use metadata tags to demo ability to group indicators in different ways.
Addresses #4, #33, and #38.

### PR DESCRIPTION
**Pull request by EFT-Defra**
Close #60 by providing a view of the indicators by theme, similar to how they are displayed in the current OIF update.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -73,12 +73,8 @@ country:
...
 # translation, but feel free to change it as needed.
 frontpage_introduction_banner:
-  title: Outcome Indicator Framework for the 25 Year Environment Plan
-  description: This is government’s annual account of how it has implemented its
-    commitments for protecting and improving the environment. It is our
-    second account of progress since we published the [25 Year
-    Environment Plan](https://www.gov.uk/government/publications/25-year-environment-plan),
-    our plan for improving the environment within a generation.
+  title: frontpage.intro_title
+  description: frontpage.intro_body
```
Comment (EFT-Defra) [inline _config.yml:77]: Not sure why it's showing this again

File: _config.yml [hunk 2]
```diff
@@ -112,13 +108,18 @@ collections:
     output: true
   goals:
     output: true
+  themes:
+    output: true
+    sort_by: code
```
Comment (EFT-Defra) [inline _config.yml:113]: add themes to collections

File: _config.yml [hunk 3]
```diff
@@ -112,13 +108,18 @@ collections:
...
 
 # Menu
 menu:
   # Use these to customise the main navigation.
   - path: /goals
-    # The "translation_key" refers to the key in the SDG Translations repository.
     translation_key: general.goals
+    # The "translation_key" refers to the key in the SDG Translations repository.
+  - path: /themes
+    translation_key: Themes
```
Comment (EFT-Defra) [inline _config.yml:122]: add a "Themes" item to the menu

File: _themes/index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                        </span>
+                    </div>
+                </div>
+            </div>
+        {% endfor %}
+    {% endfor %}
+    </div> 
+</div>
+
+{% include footer.html %}
```
Comment (EFT-Defra) [inline _themes/index.html:67]: Basically, what I've done here is taken the html template for the indicators-by-target-for-goal page and adapted it to show indicators-by-theme instead.

File: _themes/Air.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
+name: Air
+code: A
+---
```
Comment (EFT-Defra) [inline _themes/Air.md:4]: This is how you tell the templating engine what the themes are, so there's one for each theme.

File: _themes/index.html (hunk 4 above)
Comment (TimAshelford) [reply _themes/index.html:67]: Makes sense!

**Review by TimAshelford — APPROVED** [review]

**Pull request merged**